docs/reference-guides/core-blocks.md

```diff
@@ -10,7 +10,7 @@ This page lists the blocks included in the block-library package.
 
 ## Accordion
 
-Displays a group of accordion headings and associated expandable content. ([Source](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/accordion))
+Displays a group of collapsible headings and associated expandable content. ([Source](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/accordion))
```

Comment:
Nice and simple.

I wonder if it should actually be the word "headings", since semantically it's a span. Perhaps "titles"?